pkg/utils/grub.go

```diff
@@ -18,21 +18,22 @@ package utils
 
 import (
 	"fmt"
+	agentConfig "github.com/kairos-io/kairos-agent/v2/pkg/config"
+	fs2 "github.com/kairos-io/kairos-agent/v2/pkg/utils/fs"
```

Comment:
maybe fsutils would fit better than fs2

Reply:
Good catch, wrong rename when moving the package